src/components/tasks/card/index.tsx

```diff
@@ -38,8 +34,23 @@ const Card: FC<Props> = ({ content }) => {
   const iconHeight = '25px';
   const iconWidth = '25px';
 
+  function isTaskOverdue() {
+    const cardClassNames = [classNames.card];
+    const currentDate = new Date();
+    if (status !== COMPLETED) {
+      if (localEndsOn.valueOf() - currentDate.valueOf() <= 0) {
+        cardClassNames.push(classNames.overdueTask);
+        return true;
```

Comment:
`cardClassNames` seems to be a variable local to the `isTaskOverdue()`. What do we gain by setting it?

Reply:
cardClassNames adds the "overdue-task" class name to the already existing class name of the overdue tasks div tag. It's an array that already holds the current class name of the div tag and if overdue, joins the overdue-task class name. Then the css for overdue task is added to it.

Reply:
`isTaskOverdue()` is a getter function. It should only return a value and not perform any [side-effects](https://en.wikipedia.org/wiki/Side_effect_(computer_science))

Based on what the return value is, we should manipulate the array you pushed to, not while getting the boolean state

Reply:
Have made the necessary changes. Requesting re-evaluation 😁